src/routes/gh_404.tsx

```diff
@@ -0,0 +1,15 @@
+import { encodeGitHubPagesUrl } from "../utils/gh_routing";
```

Comment:
Compare to https://github.com/allen-cell-animated/website-3d-cell-viewer/blob/main/public/gh-reroute/404.html